apps/game/src/edit/traffic_signals/mod.rs

```diff
@@ -355,6 +355,14 @@ impl State<App> for TrafficSignalEditor {
                         );
                     }
                 }
+                "Change crosswalks" => {
+                    // TODO Probably need to follow everything Cancel does
```

Comment:
Don't make edits to a traffic signal and then click change crosswalks; who knows what will happen :)